src/System.Linq/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Linq:
+TypesMustExist : Type 'System.Linq.Grouping<TKey, TElement>' does not exist in the implementation but it does exist in the contract.
```

Comment:
Exposed publicly in the implementation to enable reflection for data-binding scenarios.